.github/workflows/integration_test.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           npm install && npm run build && docker build -t sphinxlightning/sphinx-relay .
       - name: Checkout stack
         run: |
-          git clone -b update/proxy-tag https://github.com/stakwork/sphinx-stack.git stack
+          git clone  https://github.com/stakwork/sphinx-stack.git stack
```

Comment:
didn't realize we were still using a different branch we should get this updated to master